static/app/views/nav/primary/components.tsx

```diff
@@ -60,13 +60,26 @@ interface SidebarItemProps extends React.HTMLAttributes<HTMLLIElement> {
   children: React.ReactNode;
   label: string;
   showLabel: boolean;
+  disableTooltip?: boolean;
 }
 
-export function SidebarItem({children, label, showLabel, ...props}: SidebarItemProps) {
+export function SidebarItem({
+  children,
+  label,
+  showLabel,
+  disableTooltip,
+  ...props
+}: SidebarItemProps) {
   const {layout} = useNavContext();
   return (
     <IconDefaultsProvider legacySize={layout === NavLayout.MOBILE ? '16px' : '21px'}>
-      <Tooltip title={label} disabled={showLabel} position="right" skipWrapper delay={0}>
+      <Tooltip
+        title={label}
+        disabled={showLabel || disableTooltip}
```

Comment:
While I was modifying the user dropdown, disabled the tooltip since it is unnecessary